src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web/Views/Apprentice/Details.cshtml

```diff
@@ -50,24 +73,33 @@
 
         <table class="govuk-table">
             <tbody class="govuk-table__body">
+
                 <tr class="govuk-table__row">
-                    <th scope="row" class="govuk-table__header das-table-cell-width-50">Employer</th>
-                    <td class="govuk-table__cell das-table-cell-width-50">@Model.Employer</td>
+                    <th scope="row" class="govuk-table__header das-table-cell-width-40">Employer</th>
+                    <td class="govuk-table__cell das-table-cell-width-40">@Model.Employer</td>
+                    <td class="govuk-table__cell das-table-cell-width-20 govuk-table__cell--numeric">
+                        @if (Model.Status == ApprenticeshipStatus.Stopped && Model.ChangeOfEmployerEnabled)
+                        {
+                            <a id="change-employer-link" class="govuk-link" href="@Url.Action("Inform", new {Model.ProviderId, Model.ApprenticeshipHashedId})">Change</a>
```

Comment:
This comment doesn't really relate to your branch, but now would probably be a good time to fix it. The route for the inform page should be: `{base}/{providerId}/apprentices/{apprenticeshipId}/change-employer`

and not `{base}/{providerId}/apprentices/{apprenticeshipId}/change-employer/inform`

This is my fault; it was listed on the routes page as the latter. Would you mind changing it here though? Be a good place to correct it. Your Url.Action should still work fine, the [Route] attribute just needs changing in the controller. Ta. :+1:

Reply:
Changed to:
```
        [Route("{apprenticeshipHashedId}/change-employer", Name = RouteNames.ApprenticeInform)]
```
Page title and GA tag:
```
@{
    ViewData["Title"] = "What you'll need";
    ViewBag.GaData.Vpv = "/apprentices/change-employer";
}
```